config/routes.rb

```diff
@@ -74,17 +75,13 @@
         end
       end
 
-      resources :criteria do
+      resources :criteria, except: [:index] do
         collection do
           post 'update_positions'
         end
       end
 
-      resources :rubric_criteria do
-        member do
-          delete 'destroy'
-        end
-
+      resources :rubric_criteria, only: [:index] do
```

Comment:
I checked and we need to include index, at least until we migrate the index action into the criteria controller.